eips_code/eip4750.py

```diff
@@ -29,6 +36,86 @@
 class ValidationException(Exception):
     pass
 
+# Validate EOF code.
+# Raises ValidationException on invalid code
+def validate_eof(code: bytes):
```

Comment:
Can we import this from `eip3540.py` ?

Reply:
I think so, if the same.

Reply:
Container format and validation changed comparing to 3540, that's the whole point of PR.

Maybe we could make 3540 code or part of it more generic and reuse here, but not usure if it's worth it.

Reply:
> Maybe we could make 3540 code or part of it more generic and reuse here, but not usure if it's worth it.

Container validation in EIP reference code would need to be two-part then somehow.